Source/WebCore/page/ScreenOrientation.cpp

```diff
@@ -184,7 +184,7 @@ uint16_t ScreenOrientation::angle() const
     auto* manager = this->manager();
     auto orientation = manager ? manager->currentOrientation() : naturalScreenOrientationType();
 
-    // https://w3c.github.io/screen-orientation/#dfn-screen-orientation-values-table
+    // https://w3c.github.io/screen-orientation/#the-current-screen-orientation-type-and-angle
```

Comment:
Doesn't this function in WebPageProxyIOS.mm also need to be fixed?
```
WebCore::ScreenOrientationType WebPageProxy::toScreenOrientationType(IntDegrees angle)
{
    if (angle == -90)
        return WebCore::ScreenOrientationType::LandscapeSecondary;
    if (angle == 180)
        return WebCore::ScreenOrientationType::PortraitSecondary;
    if (angle == 90)
        return WebCore::ScreenOrientationType::LandscapePrimary;
    return WebCore::ScreenOrientationType::PortraitPrimary;
}
```

Reply:
The above looks right to me... The above matches what Firefox and Chrome report.

Reply:
Ok, yeah... I thought the angles were the same on Android and iOS... but they are flipped 🫠